packages/apollo-server-core/src/plugin/schemaReporting/schemaReporter.ts

```diff
@@ -117,11 +117,13 @@ export class SchemaReporter {
       const result = await this.reportServerInfo(sendNextWithExecutableSchema);
       switch (result.kind) {
```

Comment:
If you'd like to factor this one out into its own PR we could get it merged fast!

Reply:
Sounds good, moved this change to #5222 🙂